scripts/main.sh

```diff
@@ -44,7 +46,16 @@ op_get_session() {
 }
 
 get_op_items() {
+  local cache_file="/tmp/tmux-op-items"
 
+  if [ -e $cache_file ]; then
```

Comment:
Use the "newer" variant:

```suggestion
  if [[ -e $cache_file ]]; then
```